src/librustc/session/config.rs

```diff
@@ -1816,7 +1816,7 @@ pub fn rustc_optgroups() -> Vec<RustcOptGroup> {
         ),
         opt::opt_s("", "sysroot", "Override the system root", "PATH"),
         opt::multi("Z", "", "Set internal debugging options", "FLAG"),
-        opt::opt_s(
+        opt::multi_s(
```

Comment:
I feel like this is abusing the `multi` argument config. While `getopts` doesn't support this kind of "pick last" feature, maybe it would make more sense to teach it about it than work around it here by suggesting that giving an argument multiple times makes sense.

Reply:
We could also stop using `getopts`, it's not that great.

Reply:
I think we'd either have to
- duplicate the logic and create a separate `multi*_last` which only differs by returning optionally a single element (picked last) or
- introduce some kind of strategy to the `multi*` but different strategies might have different requirements and return different values (single or multiple)

which I think is more trouble than it's worth in this case?